tests/unit/test_micro_agents.py

```diff
@@ -64,9 +62,7 @@ def test_coder_agent_without_summary():
     """
     mock_llm = MagicMock()
     content = json.dumps({'action': 'finish', 'args': {}})
-    mock_llm.do_completion.return_value = {
-        'choices': [{'message': {'content': content}}]
-    }
+    mock_llm.completion.return_value = {'choices': [{'message': {'content': content}}]}
```

Comment:
This change makes me wonder if the linting didn't run?

Reply:
I think it did and it made it a single line because now the line is smaller. (less characters)

Reply:
Ah, makes sense: my commits never touched _that_ test file, so you were the first to do so since this "hidden" linting issue was brought in. Sweet, tiny, nit. 🤣